src/test/resources/domain-events/gdpr-completed.json

```diff
@@ -0,0 +1 @@
+{"eventType":"probation-case.deleted.gdpr","version":1,"occurredAt":"2025-04-15T09:49:55.560241+01:00","description":"A case has been deleted in NDelius under GDPR rules","personReference":{"identifiers":[{"type":"CRN","value":"X028318"}]}}
```

Comment:
You're not using these JSON files anywhere at the moment - would it be simpler to leave them out for now?